### ISSUE TITLE
Open datasets with physio data

### ISSUE THREAD

**Issue by tsalo** [physiopy/physiopy.github.io#19]
This stems from https://github.com/physiopy/physiopy.github.io/issues/11#issuecomment-753409815_. I was thinking we can use this issue to build a list of datasets with physio data.

**Comment by eurunuela** [comment]
I believe I created a blank page for physio data in #16 

Should we make it a table of some kind? Maybe with name, link, citation, and brief description?

**Comment by tsalo** [comment]
I didn't realize you had already started on it!

I think that would be amazing. There are definitely more datasets with physio than those with multi-echo, so a bulleted list like we have in `tedana`'s docs probably isn't as appropriate.

**Comment by eurunuela** [comment]
> I didn't realize you had already started on it!

Yeah... it's been there for a couple of months now I believe. I'm guessing folks are either busy or focused on phys2denoise.

> I think that would be amazing. There are definitely more datasets with physio than those with multi-echo, so a bulleted list like we have in `tedana`'s docs probably isn't as appropriate.

Ideally, we could have a Google Spreadsheet embedded into the page. I think that would be the most comfortable solution for anyone willing to contribute. And it should be pretty straightforward as markdown can render HTML code.

**Comment by eurunuela** [comment]
I've just pushed a commit to #16 with the embedded spreadsheet. The spreadsheet is here: https://docs.google.com/spreadsheets/d/1i4CMTsH1JWHS8n0vY_QAYbHN_TMAT8z4tOFyzOHlSgo/edit?usp=sharing

It is supposed to update the website every time the sheet is updated.
